src/xacro/__init__.py

```diff
@@ -94,6 +94,18 @@ def abs_filename_spec(filename_spec):
     return filename_spec
 
 
+class YamlDictWrapper(object):
+    """Wrapper class providing dotted access to dict items"""
+    def __init__(self, d):
+        self.__d = d
+
+    def __getattr__(self, item):
+        result = self.__d.__getitem__(item)
+        return YamlDictWrapper(result) if isinstance(result, dict) else result
+
+    def __getitem__(self, item):
+        return self.__d.__getitem__(item)
+
```

Comment:
Did you have a particular reason to not just `__getitem__ = __getattr__`?

Reply:
No, I just didn't think about that. This would solve the "problem" you mentioned of course. I'm fine with this change.

Reply:
I'll check how that works with non-existing keys. I remember there being some difference in the behaviour between `__getitem__` and `__getattr__`.